Dockerfile

```diff
@@ -1,3 +1,9 @@
-FROM openjdk:11-jre
-COPY example/build/libs/example*.jar app.jar
-ENTRYPOINT ["java", "--add-opens", "java.base/java.lang=ALL-UNNAMED", "-jar", "/app.jar"]
\ No newline at end of file
+FROM gradle:jdk11
+COPY --chown=gradle:gradle . /mms
+WORKDIR /mms
+RUN ./gradlew bootJar
+
+RUN cp /mms/example/build/libs/example*.jar /app.jar
+ENTRYPOINT ["java", "--add-opens", "java.base/java.lang=ALL-UNNAMED","-jar", "/app.jar"] 
+EXPOSE 8080
+EXPOSE 5432
```

Comment:
don't need expose 5432
also fyi, the expose 8080 is not really needed the way we're using this, but still good to have
https://we-are.bookmyshow.com/understanding-expose-in-dockerfile-266938b6a33d